packages/orchestration/test/supports.ts

```diff
@@ -86,6 +86,13 @@ export const commonSetup = async t => {
       bld,
       ist,
     },
+    commonPrivateArgs: {
+      agoricNames,
+      localchain,
+      orchestrationService: orchestration,
+      storageNode: storage.rootNode,
```

Comment:
hm. passing the whole storage root to a contract shouldn't be common

I'm not sure what to suggest

Reply:
I expect it'll change when we test writes to vstorage